src/main/java/org/openrewrite/java/dropwizard/method/AddMissingAbstractMethods.java

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.dropwizard.method;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.JavaType;
+import org.openrewrite.java.tree.TypeTree;
+import org.openrewrite.java.tree.TypeUtils;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+import static org.openrewrite.java.dropwizard.method.util.MethodStubCreator.buildMethodStub;
+import static org.openrewrite.java.tree.Flag.Abstract;
+import static org.openrewrite.java.tree.Flag.Default;
+import static org.openrewrite.java.tree.TypeUtils.asFullyQualified;
+import static org.openrewrite.java.tree.TypeUtils.isAssignableTo;
+
+public class AddMissingAbstractMethods extends Recipe {
```

Comment:
Similarly, and optionally, we already have [AddMissingMethodImplementation](https://github.com/openrewrite/rewrite-migrate-java/blob/82d3e81627241ef77a09e8bc4bb85ebe0d736277/src/main/java/org/openrewrite/java/migrate/AddMissingMethodImplementation.java#L32), which is a little more explicit in it's use, but then saves us having to maintain two recipes for similar functionality. Please let me know your thoughts!